syntax = "proto3";

package cafenow.grpc;

service CafenowAgent {
	rpc handleProcess(BasicVO) returns (BasicVO) {}
}

message BasicVO{
	string api_type = 1;
	string message = 2;
}
//$ python -m grpc_tools.protoc -I./ --python_out=. --grpc_python_out=. ./cafenow.proto